cellengine/resources/experiment.py

```diff
@@ -119,8 +219,15 @@ def active_compensation(self) -> Compensation:
         )
 
     @active_compensation.setter
-    def active_compensation(self, compensation: str):
-        self._properties["activeCompensation"] = compensation
+    def active_compensation(self, compensation: Union[Compensation, int]):
+        if type(compensation) is Compensation:
+            self._active_comp = compensation._id  # type: ignore
```

Comment:
this and line 230R: `_active_comp` is `Union[int, str]`, so this should pass without the `# type: ignore`, right?

Reply:
:+1: